src/main/java/com/google/sps/data/Video.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+import java.io.IOException;
+import java.util.List;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+
+public class Video {
+
+  private final List<String> comments;
+  private final float commentScore;
+
+  public Video(List<String> comments/*, List<String> captions*/) throws IOException {
+    // Can add captions functionality later on.
+    this.comments = comments;
+    this.commentScore = calculateSentimentScore(String.join(". ", comments));
+  }
+
+  // Calculates sentiment score of text. The score is from -1 (negative) to +1 (positive).
+  private float calculateSentimentScore(String text) throws IOException {
+    Document doc =
+        Document.newBuilder().setContent(text).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float score = sentiment.getScore();
+    languageService.close();
+
+    return score;
+  }
```

Comment:
You made a comment about how the API only takes one string to analyze. We had discuss that merging all comments might give you a different sentiment because it might be reading it as one blob of text from a singular person than if you gave call the API once per comment and then say averaged the score. To make multiple calls to the sentiment API efficiently, you would need to call the API asynchronously using Java Threads or Futures. I would recommend Java 8 Futures. Since threads and Future might be a lot to add for this PR, I would accept the merging comments implementation for now but expect this to be improved in the next PR after researching on how to use Futures.